### PR TITLE
Feature/drivers modules

### PR DESCRIPTION
**Pull request by OliviaOstwald-NOAA**
## Pull Request Testing ##

- [X] Describe testing already performed for this Pull Request:</br>

- [X] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [X] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [X] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [  ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by ShelleyMelchior-NOAA** [comment]
@OliviaOstwald-NOAA   -- awesome! Looks good upon quick glance.

**Comment by ShelleyMelchior-NOAA** [comment]
Hi @OliviaOstwald-NOAA ! I have cloned your fork and checked out `feature/drivers-modules`. I will test your PR and let you know as I work through the steps (stats and plots for global and regional).

File: dev/drivers/scripts/hurricane/plots/jevs_hurricane_global_det_tcgen_plots.sh [hunk 1]
```diff
@@ -9,46 +9,31 @@
...
 export envir=dev
 export cyc=00
 export job=jevs_hurricane_global_det_tcgen_plots_${cyc}
 
+############################################################
+# Load modules
+############################################################
+module reset
+source ${HOMEevs}/modulefiles/${COMPONENT}/${COMPONENT}_${STEP}.sh
+module load geos/${geos_ver}
```
Comment (ShelleyMelchior-NOAA) [inline dev/drivers/scripts/hurricane/plots/jevs_hurricane_global_det_tcgen_plots.sh]: @OliviaOstwald-NOAA , let's get lines 34 and 35 out of the driver script and into the modules file.
`module load geos/${geos_ver}`
`module load proj/${proj_ver}`

**Comment by ShelleyMelchior-NOAA** [comment]
I have run the stats step for all drivers. Please review the results and then I will move onto plots.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/hurricane`
output log files:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/ jevs_hurricane_global_det_tcgen_stats.o80923095`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_hurricane_global_ens_tropcyc_stats.o80923210`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_hurricane_global_det_tropcyc_stats.o80923208`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_hurricane_regional_tropcyc_stats.o80927329`

**Comment by OliviaOstwald-NOAA** [comment]
All four stats log files look good

File: dev/drivers/scripts/hurricane/plots/jevs_hurricane_global_det_tcgen_plots.sh (hunk 1 above)
Comment (OliviaOstwald-NOAA) [reply dev/drivers/scripts/hurricane/plots/jevs_hurricane_global_det_tcgen_plots.sh]: done!

**Comment by ShelleyMelchior-NOAA** [comment]
I have completed the plots jobs. Please review the png files and the log files.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/hurricane`
log files:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_hurricane_global_det_tcgen_plots.o80950313`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_hurricane_global_det_tropcyc_plots.o80950358`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_hurricane_global_ens_tropcyc_plots.o80950945`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_hurricane_global_det_tcgen_plots.o80950313`

**Comment by OliviaOstwald-NOAA** [comment]
All log and png files look good!

**Review by ShelleyMelchior-NOAA — COMMENTED** [review]
All code changes look solid and as expected. Please do another sync fork to `feature/drivers-modules` to pull in the latest from the auth EVS develop.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]

**Pull request merged**